core/templates/pages/library-page/library-page.component.ts

```diff
@@ -56,25 +56,31 @@ export class LibraryPageComponent {
   isAnyCarouselCurrentlyScrolling: boolean = false;
   CLASSROOM_PROMOS_ARE_ENABLED: boolean = false;
   tileDisplayCount: number = 0;
-  activeGroupIndex: number;
-  libraryGroups: SummaryDict[];
-  mobileLibraryGroupsProperties: mobileLibraryGroupProperties[];
   leftmostCardIndices: number[] = [];
-  currentPath: string;
-  pageMode: string;
   LIBRARY_PAGE_MODES = LibraryPageConstants.LIBRARY_PAGE_MODES;
-  bannerImageFilename: string;
-  bannerImageFileUrl: string;
-  groupName: string;
-  activityList: ActivityDict[];
-  groupHeaderI18nId: string;
-  activitiesOwned = {
+  activitiesOwned: {[key: string]: {[key: string]: boolean}} = {
     explorations: {},
     collections: {}
   };
 
-  libraryWindowIsNarrow: boolean;
-  resizeSubscription: Subscription;
+  // These properties are initialized using Angular lifecycle hooks
+  // and we need to do non-null assertion, for more information see
+  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
+
+  resizeSubscription!: Subscription;
+  // The following property will be assigned null when user
+  // has not selected any active group index.
+  activeGroupIndex!: number | null;
+  activityList!: ActivityDict[];
+  bannerImageFilename!: string;
+  bannerImageFileUrl!: string;
+  currentPath!: string;
+  groupName!: string;
+  groupHeaderI18nId!: string;
+  libraryGroups!: SummaryDict[];
+  libraryWindowIsNarrow!: boolean;
+  mobileLibraryGroupsProperties!: mobileLibraryGroupProperties[];
```

Comment:
```suggestion
  mobileLibraryGroupsProperties!: MobileLibraryGroupProperties[];
```

modify the interface definition

Reply:
Done.